docs/sources/mimir/configure/about-versioning/index.md

```diff
@@ -0,0 +1,913 @@
+---
+aliases:
+  - ../configuring/about-versioning/
+description: Learn about guarantees for this Grafana Mimir major release.
+menuTitle: Versioning
+title: About Grafana Mimir versioning
+weight: 50
+---
+
+# About Grafana Mimir versioning
+
+This topic describes our guarantees for this Grafana Mimir major release.
```

Comment:
```suggestion
  - ../configuring/about-versioning/
description: Understand what major-release features are supported, unsupported, or experimental in Grafana Mimir.
menuTitle: Major-release features
title: Understand major-release features available in Mimir
weight: 50
---

# Understand major-release features available in Mimir

The features that follow are available in this major release of Mimir.
```

The file (and file name) seem to be inaccurate. Rather than covering a topic such as _Semantic versioning_, this page covers the state of feature coverage for a given major release.